px-map-behavior-layer-IMS.es6.js

```diff
@@ -243,31 +243,43 @@
         this._addEditableTools(mapInst, IMSLayer);
       }
 
-      //Make request to IMS to get collection
-      this.url = `/v1/collections/${options.layerName}/spatial-query/bbox-interacts?`+
-        `left=${initialBounds._southWest.lng}&right=${initialBounds._northEast.lng}&top=${initialBounds._northEast.lat}&bottom=${initialBounds._southWest.lat}`;
-      if(options.demo) this.url = 'demo/px-map-layer-geojson-data.json';
-      this.querySelector('#get-collection').generateRequest();
+      const layerStartingZoomValue = this._getLayerStartingZoomValue();
+      if(this.parentNode.elementInst.getZoom() >= layerStartingZoomValue) {
```

Comment:
You are only using `layerStartingZoomValue` once in this scope, so you may as well not assign it and do:
`if(this.parentNode.elementInst.getZoom() >= this._getLayerStartingZoomValue()) {`

Also while talking about _getLayerStartingZoomValue(), it should be able to deal with the case where just a single numbe is set (e.g. '18' and not '18+'). Just need to alter the `end` const's logic.